packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyTable.tsx

```diff
@@ -31,11 +30,7 @@ interface CalculatedPropertyTableProps {
   iModelId: string;
   mappingId: string;
   groupId: string;
-  setSelectedCalculatedProperty: React.Dispatch<
-  React.SetStateAction<
-  CreateTypeFromInterface<CalculatedPropertyType> | undefined
-  >
-  >;
+  setSelectedCalculatedProperty: React.Dispatch<React.SetStateAction<CreateTypeFromInterface<CalculatedPropertyType> | undefined>>;
```

Comment:
Not for this PR. I just noticed that this is wrapped twice.

Reply:
What do you mean by "wrapped twice"? All I did here was put the whole generic type into a single line, because VSCode kept indenting the generic types and the linter kept failing because of that.

Reply:
Its unrelated to this PR. I just noticed that CreateTypeFromInterface is being used on CalculatedPropertyType which does nothing. This change made me notice it.